### PR TITLE
feat(): add tutorial intro screen

### LINKED ISSUE #1318: Very clear way to exit tutorial once it's started

### PR DESCRIPTION
**Pull request by jacobnisnevich**
Resolves #1318 

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,8 +26,11 @@
     "test:lint:es": "eslint src test",
     "test:lint:ts": "tslint --project tsconfig.json 'src/**/*.{ts,tsx}' 'test/**/*.{ts,tsx}'",
     "test:types": "tsc --skipLibCheck --noEmit",
-    "test:unit": "jest",
-    "test:unit:coverage": "jest --coverage",
+    "test:integration": "jest integration",
+    "test:integration:coverage": "yarn test:integration --coverage",
```
Comment (jacobnisnevich) [inline package.json:30]: This one probably doesn't make much sense 🤔

File: src/common/components/game/TutorialIntroScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  public render(): JSX.Element {
+    return (
+      <Dialog
+        open={this.state.open}
+        className={this.props.classes.modal}
+      >
+        <DialogTitle>Wordbots Gameplay Tutorial</DialogTitle>
+        <DialogContent>
+          <p>Welcome to the Wordbots Tutorial!</p>
+          <p>This tutorial should take just 3 to 5 minutes to complete and will teach you all the basic gameplay mechanics required to play the game of Wordbots, including how to place robots and structures onto the board, how to use events to tilt the game in your favor, and the basic steps that go into each turn of Wordbots.</p>
```
Comment (AlexNisnevich) [inline src/common/components/game/TutorialIntroScreen.tsx:36]: nit: I'd make some small copy changes:
- "robots and structures" -> "robots"
– "into each turn of Wordbots" -> "into each turn"

File: src/common/components/game/GameArea.tsx [hunk 3]
```diff
@@ -279,6 +280,7 @@ export default class GameArea extends React.Component<GameAreaProps, GameAreaSta
             winnerName={(winner && winner !== 'draw') ? usernames[winner] : null}
             onClick={onClickEndGame}
           />
+          {tutorialStep && tutorialStep.idx === 0 && <TutorialIntroScreen onClickEndGame={onClickEndGame} />}
```
Comment (AlexNisnevich) [inline src/common/components/game/GameArea.tsx:283]: Ah, so we're not using `tutorialScript[0]` in `tutorial.ts` at all now? Maybe you should just get rid of the first tutorial step completely or make it clear that it's unused now.

File: package.json (hunk 1 above)
Comment (AlexNisnevich) [reply package.json:30]: Yeah we can leave this one out for now.

**Review by AlexNisnevich — APPROVED** [review]
Looks good, left some comments.

File: src/common/components/game/GameArea.tsx (hunk 3 above)
Comment (jacobnisnevich) [reply src/common/components/game/GameArea.tsx:283]: Oh we still are. This was just my way of checking that we're at the beginning of the tutorial

**Pull request merged**